<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Магазин товаров</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">

    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>Доступные товары</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:16pt; font-weight:bold;</string>
      </property>
     </widget>
    </item>

    <item>
     <widget class="QTableWidget" name="tableProducts">
      <column>
       <property name="text">
        <string>Название</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Цена</string>
       </property>
      </column>
     </widget>
    </item>

    <item>
     <widget class="QLabel" name="labelMessage">
      <property name="text">
       <string></string>
      </property>
      <property name="styleSheet">
       <string notr="true">color:green;</string>
      </property>
     </widget>
    </item>

    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutBuyUser">
      <item>
       <widget class="QPushButton" name="btnBuy">
        <property name="text">
         <string>Купить выбранный товар</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="inputBuyQuantityUser">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <item>
     <widget class="QPushButton" name="btnLogoutUser">
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
    </item>

   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>